### ISSUE TITLE
Failed to start zigbee: RaspBee/Conbee with homeassistant OS (Supervised) on Raspberry 4 (2GB)

### ISSUE THREAD

**Issue by pumuugel82** [zigbee2mqtt/hassio-zigbee2mqtt#35]
**Description of problem:**
Cannot start zigbee2mqtt with desden-elektornik's RaspBee on homeassistant OS (Superviced)

**Zigbee2mqtt add-on version (if edge, please report commit hash):**
```
Current version: 1.17.1-4 
```

**Your entire configuration from the frontend (with sensitive fields redacted)**:
```
data_path: /config/zigbee2mqtt
external_converters: []
devices: devices.yaml
groups: groups.yaml
homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://homeassistant'
  user: replaced
  password: replaced
serial:
  port: /dev/ttyAMA0
  adapter: deconz
advanced:
  log_level: warn
  pan_id: 6754
  channel: 11
  network_key:
    - 1
    - 3
    - 5
    - 7
    - 9
    - 11
    - 13
    - 15
    - 0
    - 2
    - 4
    - 6
    - 8
    - 10
    - 12
    - 13
  availability_blocklist: []
  availability_passlist: []
device_options: {}
blocklist: []
passlist: []
queue: {}
frontend:
  port: 8099
experimental: {}
socat:
  enabled: false
  master: 'pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777'
  slave: 'tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5'
  options: '-d -d'
  log: false
zigbee_herdsman_debug: true

```

**Running platform (details from Suypervisor > System)**
```
Core:
Version core-2021.2.3
Newest Version core-2021.2.3 

Supervisor:
Version supervisor-2021.02.9
Newest Version supervisor-2021.02.9 

Host:
IP Address 192.168.178.55/24
Operating System Home Assistant OS 5.11 
```

**Your logs from Home Assistant Supervisor**
```
[15:47:04] INFO: Zigbee Herdsman debug logging enabled
[15:47:05] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.17.1 start /app
> node index.js
2021-02-18T14:47:09.131Z zigbee-herdsman:adapter Failed to validate path: 'Error: spawn udevadm ENOENT'
2021-02-18T14:47:09.139Z zigbee-herdsman:deconz:driver Set delay to 300
CREATED DECONZ ADAPTER
2021-02-18T14:47:09.144Z zigbee-herdsman:controller:log Starting with options '{"network":{"networkKeyDistribute":false,"networkKey":[1,3,5,7,9,11,13,15,0,2,4,6,8,10,12,13],"panID":6754,"extendedPanID":[221,221,221,221,221,221,221,221],"channelList":[11]},"serialPort":{"path":"/dev/ttyAMA0","adapter":"deconz"},"databasePath":"/config/zigbee2mqtt/database.db","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","backupPath":"/config/zigbee2mqtt/coordinator_backup.json","adapter":{"concurrent":null,"delay":null}}'
2021-02-18T14:47:09.146Z zigbee-herdsman:deconz:driver Opening with /dev/ttyAMA0
2021-02-18T14:47:09.156Z zigbee-herdsman:deconz:driver Serialport opened
2021-02-18T14:47:09.159Z zigbee-herdsman:controller:log Started with result 'resumed'
2021-02-18T14:47:09.159Z zigbee-herdsman:controller:log Injected database: true, adapter: true
2021-02-18T14:47:09.240Z zigbee-herdsman:deconz:driver send read parameter request from queue. seqNr: 1 paramId: 1
2021-02-18T14:47:12.144Z zigbee-herdsman:deconz:adapter checking coordinator simple descriptor
2021-02-18T14:47:12.151Z zigbee-herdsman:deconz:driver DATA_REQUEST - destAddr: 0x0 EP:0 SeqNr. 2 request id: 1
2021-02-18T14:47:20.139Z zigbee-herdsman:deconz:driver Timeout for request - CMD: 0xa seqNr: 1
Zigbee2MQTT:error 2021-02-18 15:47:20: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-02-18 15:47:20: Failed to start zigbee
Zigbee2MQTT:error 2021-02-18 15:47:20: Exiting...
Zigbee2MQTT:error 2021-02-18 15:47:20: undefined
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.17.1 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.17.1 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-02-18T14_47_20_216Z-debug.log
```

**Comment by mad2xlc** [comment]
Try to replace your port with 
port: /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2412023-if00

Verify your path with the log Supervisor -> Host -> Hardware

**Comment by pumuugel82** [comment]
Below there is an extract from the hardware output. I think it shows the right thing. I don't have anything like "/dev/serial/by-id/..." 

As the RaspBee is directly attached to the GPIO Pins, I wonder if its possible to configure UART somehow as I have to do with Raspberry PI OS as mentioned here [deconz-docker-instructions](https://github.com/marthoc/docker-deconz#configuring-raspbian-for-raspbee). But I don't see how to do this with Home-Assistant OS 5.

Extract from hardware:
```
 - name: ttyAMA0
    sysfs: /sys/devices/platform/soc/fe201000.serial/tty/ttyAMA0
    dev_path: /dev/ttyAMA0
    subsystem: tty
    by_id: null
    attributes:
      DEVLINKS: /dev/serial1
      DEVNAME: /dev/ttyAMA0
      DEVPATH: /devices/platform/soc/fe201000.serial/tty/ttyAMA0
      MAJOR: '204'
      MINOR: '64'
      SUBSYSTEM: tty
      TAGS: ':systemd:'
      USEC_INITIALIZED: '2568292'
```

**Comment by pumuugel82** [comment]
Well sometimes talking to others of the same domain just helps. 

Before opening this issue I ran HA with an USB CC2531 Chip with antanna but due to poor performance I now try to replace the adapter under the hood with RaspBee which I own already. Nevertheless I use the same RPi with the same HA OS installation. That's the situation. As mentioned above simply removing the cc2531 USB Stick and plugin the RaspBee into GPIO didn't work.

Additionally I had to remove the microSD card, mount it in another machine with Linux. There, mount the first partition (FAT32) from the micoSD and edit the "config.txt". Add the line: `dtoverlay=disable-bt` in order to disable Bluetooth. Save + put it back into RPi and start. Now everything works as expected. Opposed to the referenced issue below (they have port: /dev/ttyS0), serial is still set to and it works:

```
serial:
  port: /dev/ttyAMA0
  adapter: deconz
```

This issue helped a lot https://github.com/home-assistant/operating-system/issues/1006.

Thank you guys for reading and helping.
